validator/.idea/vcs.xml

```diff
@@ -1,5 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="IssueNavigationConfiguration">
```

Comment:
is this supposed to be in the PR? or in .gitignore?

Reply:
There's no hard-and-fast rule. .idea files are part of the IntelliJ suite and are generally useful to have assuming everybody is using IntelliJ. Other people prefer not to have them in the repo.